docs/instructlab-cli-1.0.0.md

```diff
@@ -0,0 +1,35 @@
+# The Road to 1.0.0
+
+_Or: How I Learned to Stop Worrying and Love to GA_
+
+## Context and Goals
+
+At the time of this writing, the InstructLab CLI has gone from a scrappy research project to an upstream community serving as the basis for multiple downstreams, with the goal to continuing to evolve the community to encourage more participation from additional stakeholders. To wit, it would behoove us to determine what exactly we should be roadmapping between now and a proper 1.0.0 release, which demonstrates the following to existing and potential community members:
+
+1. An official goalpost for the community denoting the evolution of InstructLab from a pre-1.0 project lacking the stability and supportability typically seen from 1.0-and-beyond projects.
+1. A dedicated set of V1 interfaces that can be counted on for continuous usage of InstructLab 1.0 with future provisions made for backwards compatibility for subsequent Y-Streams and Z-Streams.
+1. A commitment from the Oversight Committee and Maintainer teams to continue to maintain InstructLab throughout a 1.y cycle and work towards an eventual 2.0.
+
+## MVP for an InstructLab CLI 1.0.0
+
+At a high-level, these are the items the Maintainer teams believe should serve as prereqs for releasing an InstructLab CLI 1.0.0:
+
+- Removal of the need for a `config.yaml` with a fully-realized configuration scheme, centered around the usage of system profiles
+- An official v1 REST API schema
```

Comment:
This REST API looks nice. I think this encourages people's contributions, and people with a less power laptop can train the dataset faster.